### ISSUE TITLE
Re-instate plan description

### ISSUE THREAD

**Issue by RCheesley** [OSTraining/Simplerenew#390]
Hi folks,

Great extension and super helpful for integrating Joomla with Recurly.

I note from an old issue: https://github.com/OSTraining/Simplerenew/issues/110 that you revoked the 'plan description' field.  This is something that we need quite badly for a project, to the extent that if it's not available we'll have to do some kind of workaround with modules and content articles which seems a little daft if we could have the description set in one place.

There doesn't seem to be anything by way of reasoning for it being removed other than "We don't think we need it it" and a comment was made: "Plan descriptions could be displayed someplace on the front end as part of the subscribe/change-plan screens." but I can't see anywhere that has been implemented either.

So, the feature request I'd like to make is to allow the plan description to be incorporated into SimpleRenew and available for use in the themes.

**Comment by nicksavov** [comment]
Hi Ruth,

Could you explain your use-case? There might be a better way to do what you're wanting.

**Comment by RCheesley** [comment]
We have three plans, they each have a description. On the plan select page we need to show the description.  Ideally in a fly-out so as not to clutter - e.g. select the plan and the details for that plan show on the right, select a different plan and the details for that plan are shown instead.

The most sensible place to hold this information would be in the plan description, I would have thought, in Recurly.

Thanks for the quick response.

**Comment by nicksavov** [comment]
Thanks Ruth! It won't be difficult to re-add the feature. Then your front-end designer can customize the look how you'd like it.

**Comment by RCheesley** [comment]
So should we just suck the stuff back in from the PR which removed the functionality? Will that still bring down the info from Recurly?

**Comment by nicksavov** [comment]
Yes, but that commit looks like it had additional changes, so you'd just want to keep the parts about the description. If your developer would like to send a PR for it, that would be great. Bill could get it reviewed shortly thereafter.

**Comment by OSTraining** [comment]
All sounds good - as Nick points out, not a particularly huge deal to make this happen. It will require a change in the db schema for plans.

**Comment by RCheesley** [comment]
OK awesome, will take a look and try to make a PR in the next few days unless someone beats me to it :) Thanks folks!
